deepchem/utils/data_utils.py

```diff
@@ -267,9 +268,7 @@ def load_sdf_files(input_files: List[str],
                 for task in tasks:
                     df_row.append(mol.GetProp(str(task)))
 
-            conf = mol.GetConformer()
-            positions = conf.GetPositions()
-            pos_x, pos_y, pos_z = zip(*positions)
```

Comment:
Here, `pos_x`, `pos_y`, `pos_z` will be tuples and not lists. The change makes data type of these as list.